app/views/studios/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h3>Studios</h3>
+
+<% @studios.each do |studio| %>
+  <section id=<%= "studio-#{studio.id}" %>>
```

Comment:
Nice set up with the section ID to allow for robust testing! If, on an IC, you ever get hung up on the within block syntax, feel free to implement the story/test it without within blocks and come back to it at the end if you have remaining time. Looks like you got this working, but I know some students had trouble with the syntax for within blocks with this IC!